tensorboard/webapp/plugins/npmi/npmi_component.ng.html

```diff
@@ -15,6 +15,8 @@
 limitations under the License.
 -->
 <div class="npmi-container">
-  This is a preliminary version of the nPMI plugin. It is under active
-  development.
+  <npmi-inactive *ngIf="runs.size === 0; else dataAvailable"></npmi-inactive>
```

Comment:
Hmm, I don't think this is the right condition: run selection does not care about plugin selection. Other plugins can show an inactive page when `runToTag` for a given plugin is empty.

Reply:
I currently dont have runToTag in my state as I have fixed tags that I dont really need to fetch, and I think I cannot access it from the core state, right?
I thought I would switch this later so something like checking if annotations/metrics/values present.